content/docs/configuration.md

```diff
@@ -83,7 +83,30 @@ synced_files:
     dest: .
 ```
 
+## RPM spec file not in upstream repository
 
+If you do not want to have the RPM spec file in your upstream repository, for some reason,
+you can download it in [actions](/docs/actions/) section.
+
+Add [actions](/docs/actions/) section and
+hook `post_upstream_clone` into your `.packit.yaml` configuration file.
+Add row for downloading your RPM spec file from Fedora dist-git repository
+with your favorite command.
+
+The configuration file with downloading the RPM spec file now looks:
+
+```yaml
+specfile_path: packit.spec
+synced_files:
+  - packit.spec
+  - .packit.yaml
+upstream_project_name: packitos
+downstream_package_name: packit
+actions:
+    post_upstream_clone:
+    - wget https://src.fedoraproject.org/rpms/packit/raw/master/f/packit.spec
```

Comment:
post_upstream_clone accepts str as well, no need to create a list

Reply:
Fixed by commit https://github.com/packit-service/packit.dev/pull/36/commits/e877d65fe10018eca4d19bf6d2096d3a0bad3d72